pkg/apis/agones/v1/gameserver_test.go

```diff
@@ -375,6 +375,12 @@ func TestGameServerValidate(t *testing.T) {
 				Name:       "main",
 				HostPort:   5001,
 				PortPolicy: Dynamic,
+			}, {
+				Name:       "sidecar",
+				HostPort:   5002,
+				PortPolicy: Static,
+				ContainerPort: 5002,
+				ContainerName: "anothertest",
```

Comment:
is this the only place it needs to be tested?

Reply:
Other than the comment below on splitting things up, I don't see any other unit tests that need updating at this point.

But like i mentioned in the main comment - would definitely want a e2e test.